tests/testthat/test-onevax_xvwr.R

```diff
@@ -26,16 +26,17 @@ test_that("run_onevax_xvwr works correctly", {
     expect_equal(y2[[i]]$T[length(tt), , ], y3[[i]]$T[1, , ])
   }
 
-  uptake <- c(0.5, 1)
+  primary_uptake <- booster_uptake <- 0.5
```

Comment:
```suggestion
  primary_uptake <- c(0.5, 1)
```
Keep check that we can supply as a vector - i.e. different for each param set